lib/media.js

```diff
@@ -152,8 +155,14 @@ function injectTags(fileArray, site, tag) {
   return bluebird.resolve(_.map(fileArray, function (file) {
     if (tag === STYLE_TAG) {
       return `<link rel="stylesheet" type="text/css" href="${file}${buster}">`;
+    } else if (tag === ASYNC_SCRIPT_TAG) {
+      return `<script async crossorigin="anonymous" src="${file}" type="text/javascript"></script>`;
+    } else if (tag === DEFER_SCRIPT_TAG) {
+      return `<script defer crossorigin="anonymous" src="${file}" type="text/javascript"></script>`;
+    } else if (tag === ASYNC_DEFER_SCRIPT_TAG) {
+      return `<script async defer crossorigin="anonymous" src="${file}" type="text/javascript"></script>`;
     } else {
-      return `<script type="text/javascript" src="${file}${buster}"></script>`;
+      return `<script crossorigin="anonymous" type="text/javascript" src="${file}${buster}"></script>`;
```

Comment:
What's the purpose of adding `crossorigin="anonymous"`? Will this cause unintentional CORS errors maybe? Not fully clear on what it does for us

Reply:
The attribute adds CORS headers to the request for the script itself. The browser walls off scripts fetched without explicit CORS support in weird, subtle ways, including [blocking their access to error handlers](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attr-crossorigin). It doesn't affect XHR requests that the script initiates, so it should be safe.

Since we're serving script assets from the same origin as our article pages, though, we don't need it immediately. I'd be fine with omitting it now and shipping it to our FOSS users in a future prerelease.